templates/masterclasses.html

```diff
@@ -12,6 +12,13 @@ <h1 class="">Masterclasses</h1>
           Register your session
         </a>
       </p>
+      <form class="p-search-box">
+        <label class="u-off-screen" for="search">Search</label>
+        <input type="search" id="search" class="p-search-box__input" name="search" placeholder="Search" required=""
+          autocomplete="on">
+        <button type="reset" class="p-search-box__reset"><i class="p-icon--close">Close</i></button>
+        <button type="submit" class="p-search-box__button"><i class="p-icon--search">Search</i></button>
```

Comment:
Submitting the form with enter key or pressing this button goes to a page with the ?search=<query> query param, but doesn't search. I think with a onsubmit property on the form referencing a js function to trigger the search you can prevent that.

Reply:
yeah I'll add a event.preventDefault on the search